pom.xml

```diff
@@ -7,6 +7,15 @@
     <groupId>org.example</groupId>
     <artifactId>cs6375-ann</artifactId>
     <version>1.0-SNAPSHOT</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
```

Comment:
The only concern I have is whether or not the TA will be able to build the project if we depend on third party libraries.  If we do it this way then we may need to provide an already built jar to the TA.  What do you think?

I'm fine with simply providing a jar and if he can't figure out how to get the dependencies then it's on him.

Reply:
@joshua-newhouse We can upload jar file in elearning along with code. We can update readme file to execute the jar file command